.gitignore

```diff
@@ -1,3 +1,5 @@
 .vscode/
+__pycache__/
```

Comment:
also the `tests/__pychache/`

Reply:
Kind of is kind of isnt... The __pychache__ directory isn't gitignored but all files inside of it are, this slight difference means that while nothing extra is added to git it can cause vscode to trip out